google/cloud/spanner/doc/spanner-main.dox

```diff
@@ -25,9 +25,9 @@ into your project.
 - [Client::Read()](@ref google::cloud::spanner::Client::Read()) - to read the
```

Comment:
`s/ -//` to match the form of the previous two entries.

Reply:
Fixed